### PR TITLE
Fix parsing of hostname from cmdline for ipv6.

### PR DESCRIPTION
**Pull request by rvykydal**
Resolves: rhbz#1554271
Resolves: rhbz#1546876

### CODE REVIEW

File: pyanaconda/network.py [hunk 1]
```diff
@@ -1160,13 +1160,17 @@ def ibftIface():
...
-    ipopts = flags.cmdline.get('ip')
+    ipopts = cmdline.get('ip')
     # Example (2 options):
     # ens3:dhcp 10.34.102.244::10.34.102.54:255.255.255.0:myhostname:ens9:none
     if ipopts:
         for ipopt in ipopts.split(" "):
+            if ipopt.startswith("["):
+                # Replace ipv6 addresses with empty string, example of ipv6 config:
+                # [fd00:10:100::84:5]::[fd00:10:100::86:49]:80:myhostname:ens9:none
+                ipopt = re.sub(r'\[[^\]]+]', '[]', ipopt)
```
Comment (jkonecny12) [inline pyanaconda/network.py]: Wouldn't it be better to have this regular expression compiled before in an init?

**Comment by rvykydal** [comment]
Just updated commit message BZ reference.

File: pyanaconda/network.py (hunk 1 above)
Comment (rvykydal) [reply pyanaconda/network.py]: For consistency and better readability, I'll do it.

**Comment by jkonecny12** [comment]
jenkins, test this please

**Comment by rvykydal** [comment]
jenkins, test this please

**Comment by rvykydal** [comment]
Ported to master with https://github.com/rhinstaller/anaconda/pull/2106

**Pull request merged**